env.example

```diff
@@ -213,4 +213,15 @@ JIBRI_XMPP_PASSWORD=
 #RESTART_POLICY=unless-stopped
 
 # Jitsi image version (useful for local development)
-#JITSI_IMAGE_VERSION=latest
+JITSI_IMAGE_VERSION=stable-8252
+
+# Jibri recording and livestreaming
+ENABLE_RECORDING=0
+ENABLE_LIVESTREAMING=0
+ENABLE_SUBDOMAINS=1
+JIBRI_RECORDING_RESOLUTION=1920x1080
+
+# whiteboard support
+WHITEBOARD_ENABLED=1
+WHITEBOARD_DOMAIN=whiteboard.meet.jitsi
```

Comment:
I think we should have this be similar to how we handle Excalidraw. There is the builtin option, and the external option, which requires the public URL. See `ETHERPAD_URL_BASE`.